### PR TITLE
[MOB-2820] Rename Supplier to Quotes

### PR DESCRIPTION
**Pull request by EdwardW2**
## JIRA
[MOB-2820](https://karhoo.atlassian.net/browse/MOB-2820)

## DESCRIPTION
[Create a description of what you have changed in this branch]

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [ ] New feature 
- [ ] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [x] Other: Alignment

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [x] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [x] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

File: uisdk/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -244,7 +244,7 @@
     </style>
 
     <style name="KhSupplierListItemView">
```
Comment (mohadian) [inline uisdk/src/main/res/values/styles.xml:246]: Perhaps it is better to rename this style as well

**Review by mohadian — COMMENTED** [review]
It would be good to remove the Supplier and replace it with the Quote wherever possible to avoid any confusion. 
e.g. 
In strings.xml `<!-- Supplier List -->`
In BookingQuotesViewContract.kt  `BookingSupplierWidget`, `setSupplierListVisibility` `BookingSupplierEvent` etc
the id for the UI views etc

File: uisdk/src/main/res/layout/uisdk_activity_booking_content.xml [hunk 2]
```diff
@@ -33,7 +33,7 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintTop_toBottomOf="@id/toolbar"/>
 
-    <com.karhoo.uisdk.screen.booking.supplier.SupplierListView
+    <com.karhoo.uisdk.screen.booking.quotes.QuotesListView
         android:id="@+id/supplierListWidget"
```
[the remark this answers is not in the document]
Comment (EdwardW2) [reply uisdk/src/main/res/layout/uisdk_activity_booking_content.xml]: Done

File: uisdk/src/main/res/layout/uisdk_booking_request.xml [hunk 3]
```diff
@@ -33,7 +33,7 @@
                 android:src="@drawable/uisdk_ic_close"
                 android:layout_gravity="end" />
 
-            <com.karhoo.uisdk.screen.booking.booking.supplier.BookingSupplierView
+            <com.karhoo.uisdk.screen.booking.booking.quotes.BookingQuotesView
                 android:id="@+id/bookingRequestSupplierWidget"
```
[the remark this answers is not in the document]
Comment (EdwardW2) [reply uisdk/src/main/res/layout/uisdk_booking_request.xml]: Updated

File: uisdk/src/main/res/values/styles.xml (hunk 1 above)
Comment (EdwardW2) [reply uisdk/src/main/res/values/styles.xml]: Updated

**Review by mohadian — APPROVED** [review]

**Pull request merged**